back/infolica/scripts/utils.py

```diff
@@ -37,8 +37,8 @@ def serialize_many(cls, _query):
     """ Return model record parameters """
     @classmethod
     def get_model_record_attributes(cls, record):
-        return [a for a in dir(record) if not a.startswith('_')] if record else []
-    
+        [a for a in dir(record) if not (a == 'id' or a.startswith('__') or a.startswith('_'))] if record else []
```

Comment:
Est-ce qu'on ne peut pas mettre 
[a for a in dir(record) if not a.startswith('_')] if record else []
car a.startswith('_') est plus restrictif que a.startswith('__') et comme ça on conserve l'id, qu'on n'a pas besoin de redéfinir dans la vue (ex: views/facture.py ligne 69)